.vscode/tasks.json

```diff
@@ -20,7 +20,7 @@
 	"showOutput": "silent",
 
 	// we run the custom script "compile" as defined in package.json
```

Comment:
it's a generated file from vscode (containing settings to start an extension)
I've just modified a parameter to fix the compile task